Themes/default/Admin.template.php

```diff
@@ -1635,4 +1635,94 @@ function template_admin_quick_search()
 								</span>';
 }
 
+/**
+ * This shows the policy settings
+ */
+function template_edit_policy()
```

Comment:
The layout of this template is a bit messy. But we can fix that later. :)

Reply:
Like i said,
i don't want to invest time in stuff in topic where i realy bad.

Reply:
No problem. When I said, "But we can fix that later," I meant that we can deal with it in another PR later sometime. 🙂